qcschema/dev/molecule.py

```diff
@@ -3,6 +3,10 @@
 """
 molecule = {
     "$schema": "http://json-schema.org/draft-04/schema#",
+    "name": "qc_schema_molecule",
+    "version": "dev",
+    "description": "The MolSSI Quantum Chemistry Molecular Schema",
+    "type": "object",
```

Comment:
below at https://github.com/MolSSI/QC_JSON_Schema/pull/49/files#diff-fb40813be873a1ea003de289cb1f9373R59, length isn't really `(nat,)`, right for bonds?

Reply:
Correct, fixed.